java/src/main/java/actorworkflow/CustomerLoyaltyWorkflowImpl.java

```diff
@@ -0,0 +1,147 @@
+package actorworkflow;
+
+import io.temporal.activity.ActivityOptions;
+import io.temporal.api.common.v1.WorkflowExecution;
+import io.temporal.api.enums.v1.ParentClosePolicy;
+import io.temporal.client.WorkflowExecutionAlreadyStarted;
+import io.temporal.workflow.*;
+import org.slf4j.Logger;
+
+import java.time.Duration;
+import java.util.ArrayList;
+
+public class CustomerLoyaltyWorkflowImpl implements CustomerLoyaltyWorkflow {
+
+    private static final Logger logger = Workflow.getLogger(CustomerLoyaltyWorkflowImpl.class);
+
+    private final CustomerLoyaltyActivities activities =
+            Workflow.newActivityStub(
+                    CustomerLoyaltyActivities.class,
+                    ActivityOptions.newBuilder()
+                            .setStartToCloseTimeout(Duration.ofSeconds(2))
+                            .build());
+
+    boolean accountActive = true;
+
+    Customer customer;
+
+    @Override
+    public String customerLoyalty(Customer customer) {
+        System.out.println("Customer passed in is: " + customer);
```

Comment:
should use workflow logger, no system out in workflow code

Reply:
Manual testing leftover. Deleting.